homeassistant/components/technove/sensor.py

```diff
@@ -0,0 +1,162 @@
+    """Describes TechnoVE sensor entity."""
+
+    value_fn: Callable[[TechnoVEStation], StateType]
+
+
+SENSORS: tuple[TechnoVESensorEntityDescription, ...] = (
+    TechnoVESensorEntityDescription(
+        key="voltage_in",
+        translation_key="voltage_in",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.voltage_in,
+    ),
+    TechnoVESensorEntityDescription(
+        key="voltage_out",
+        translation_key="voltage_out",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.voltage_out,
+    ),
+    TechnoVESensorEntityDescription(
+        key="max_current",
+        translation_key="max_current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.max_current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="max_station_current",
+        translation_key="max_station_current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.max_station_current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="energy_total",
+        translation_key="energy_total",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.energy_total,
+    ),
+    TechnoVESensorEntityDescription(
+        key="energy_session",
+        translation_key="energy_session",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.energy_session,
+    ),
+    TechnoVESensorEntityDescription(
+        key="rssi",
+        translation_key="rssi",
```

Comment:
Does the device have protocols other than Wi-Fi? If not, consider removing the translation key, as it will take the device class name (Signal strength)

Reply:
It only has wifi. Done.